static/temp/foods.json

```diff
@@ -0,0 +1,250 @@
+[
+  {
+    "name": "Banana ripe, slightly ripe, raw",
+    "categoryName": "Fruit and Fruit Juices",
+    "categoryIcon": "EggAltOutlinedIcon",
+    "overview": {
+      "info": "This is a cool food",
+      "NutrientScore": ["A", "B", "C", "D"],
+      "climateImpact": {
+        "title": "Climate Impact",
+        "weight": "20kg co2",
+        "weightTitleSubscript": "Per serving",
+        "equivalent": "200kg CO2 is equivalent to",
+        "data": [
+          {
+            "name": "0.2 gallons of diesel consumed",
```

Comment:
Rename `name` to `title`. This change should be for all `name` key under `comparisons`.

Reply:
done